product_website_description/views/product_template_views.xml

```diff
@@ -10,12 +10,7 @@
                     string="Product Description for Website Display"
                     name="website_display_description"
                 >
-                    <field
-                        name="website_description"
-                        widget="html"
-                        class="oe-bordered-editor"
-                        options="{'style-inline': true, 'codeview': true }"
-                    />
+                    <field name="website_description" widget="text" />
```

Comment:
@kanda999 Can you please add a comment to briefly explain why this change is made.